pkgs/development/libraries/libcint/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, openblas
+, python
+, pythonPackages
```

Comment:
generally you'll want to import only python. Then just access the packages by doing `python.pkgs`